angular/src/lib/select/select.component.ts

```diff
@@ -47,19 +57,19 @@ interface IState {
       [cdkConnectedOverlayPanelClass]="'cui-event-overlay__children'"
       [cdkConnectedOverlayHasBackdrop]="true"
       [cdkConnectedOverlayBackdropClass]="'cdk-overlay-transparent-backdrop'"
-      (backdropClick)="this.state.isOpen = false">
+      (backdropClick)="this.state.isOpen = false"
+    >
       <cui-list>
         <ng-content></ng-content>
       </cui-list>
     </ng-template>
   `,
   styles: [],
   host: {
-    class: 'cui-input-group cui-select'
-  }
+    class: 'cui-input-group cui-select',
```

Comment:
comma is needed here?

Reply:
Not needed, but we prefer dangling commas for git diffs. This one got added because I auto-formatted the file before saving.